src/routes/posts.js

```diff
@@ -161,14 +253,35 @@ const updatePost = async (req, res) => {
 		})
 	}
 }
+// Routes
 
+router.route("/query").get(queryPosts)
```

Comment:
i think the routes are being authenticated now so we should follow the example in this same file where passport is being used

Reply:
I don't know how the authentication is being handle that is why i didn't include it since it was included while I was still on this task.
When i picked up the task, There was only articles route, I don't know who or what the feed route is for

Reply:
yea makes sense. let's stick to the scope you were already tackling